CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Optimizely Python SDK Changelog
 
+## 3.9.2
+September 16th, 2021
+
+### New Features
+* Added new public properties to OptimizelyConfig. 
+  - sdk_key and environment_key [#338] (https://github.com/optimizely/python-sdk/pull/338)
+  - attributes and events [#339] (https://github.com/optimizely/python-sdk/pull/339)
+  - experiment_rules, delivery_rules, audiences and audiences in OptimizelyExperiment 
+    - [342] (https://github.com/optimizely/python-sdk/pull/342)
+    - [351] (https://github.com/optimizely/python-sdk/pull/351/files)
+* For details please refer to our documentation page:
+  - Python-sdk: [https://docs.developers.optimizely.com/full-stack/docs/optimizelyconfig-python]
+
+### Bug Fixes:
+* Fix event processor negative timeout interval when retrieving events from queue. [#356] (https://github.com/optimizely/python-sdk/pull/356)
```

Comment:
@Mat001 Just wanted to get you thoughts on this comment for the bug fix you added in